ui/src/app/pages/profile/demographic-survey-panel.tsx

```diff
@@ -37,14 +38,25 @@ export const DemographicSurveyPanel = (props: Props) => {
         ) : (
           <div>Survey Incomplete</div>
         )}
-        <Button
-          {...{ onClick }}
-          type={'link'}
-          style={styles.updateSurveyButton}
-          data-test-id={'demographics-survey-button'}
-        >
-          {surveyCompleted ? 'Update Survey' : 'Complete Survey'}
-        </Button>
+        {enableUpdatedDemographicSurvey ? (
```

Comment:
@jmthibault79 : This will take care of whether to open the modal by using <Button {...{onClick}
or redirect to v2 demographic survey

Reply:
thanks!